assembly/Audio/Audio.ts

```diff
@@ -0,0 +1,33 @@
+// @ts-ignore
+@external('asDOM_Audio', 'initAudio')
+export declare function initAudio(src: string, id: usize): void
+// @ts-ignore
+@external('asDOM_Audio', 'pauseAudio')
+export declare function pauseAudio(id: usize): void
+// @ts-ignore
+@external('asDOM_Audio', 'playAudio')
+export declare function playAudio(id: usize): void
+
+import { HTMLElement } from "../Element/Element";
+
+export class HTMLAudioElement extends HTMLElement {
+    constructor(src: string | null = null) {
+        super()
+        if (src) initAudio(src, this.__ptr__)
+    }
+    play(): void {
+        playAudio(this.__ptr__)
+    }
+    pause(): void {
+        pauseAudio(this.__ptr__)
+    }
+    set autoplay(toggle: boolean) {
```

Comment:
Awesome changes @JairusSW! 

Here we should avoid implementing custom logic, i.e. we should not assume that autoplay means to call play().

Instead, we should do like what I did with innerHTML, and let the JS-side objects actually do the handling of the values and logic.

Reply:
Yeah. Fixed it. :)